pkg/addon-operator/operator.go

```diff
@@ -1279,10 +1291,23 @@ func (op *AddonOperator) HandleModuleHookRun(t sh_task.Task, labels map[string]s
 		op.MetricStorage.HistogramObserve("{PREFIX}module_hook_run_seconds", d.Seconds(), metricLabels)
 	})()
 
-	logEntry.Infof("Module hook start '%s'", hm.HookName)
+	isSynchronization := hm.BindingType == OnKubernetesEvent && hm.BindingContext[0].Type == types.TypeSynchronization
```

Comment:
Could `hm.BindingContext[0].Type` be equal to Group?

Reply:
No, BindingContext here is always event-specific. `"type":"Group"` is generated lately.